nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/vendor/varian/VarianData.java

```diff
@@ -696,20 +708,24 @@ public void setArraySize(int iDim, int size) {
     @Override
     public boolean isComplex(int iDim) {
         if (iDim == 0) {
-//            return isComplex;
             String s = getPar("proc");
             return !(s != null && s.equals("rft")); // proc="ft" or "lp"
         } else {
-            String ext = "";
-            if (iDim > 1) {
-                ext += iDim;
-            }
-            String s = getPar("phase" + ext);
+            String ext = String.valueOf(iDim);
+            String s = getPar("proc"+ext);
+            boolean notRFT =  !(s != null && s.equals("rft")); 
```

Comment:
This is a bit confusing to read, I think this would be equivalent and will still check the null value
```suggestion
            boolean notRFT =  !"rft".equals(s); 
```